sdl ld error

**go.mod**
```
module game

go 1.21.3

require github.com/gen2brain/raylib-go/raylib v0.0.0-20231104153637-1e13bd47a649
```

**main.go**
```golang
package main

import rl "github.com/gen2brain/raylib-go/raylib"

func main() {
        rl.InitWindow(1280, 720, "Game")
        defer rl.CloseWindow()

        for !rl.WindowShouldClose() {
                rl.BeginDrawing()
                rl.ClearBackground(rl.White)
                rl.EndDrawing()
        }
}

```

Running `go build -tags sdl` results in the following error:
```
# game
/usr/lib/go/pkg/tool/linux_amd64/link: running gcc failed: exit status 1
/usr/bin/ld: /tmp/go-link-3620036020/000016.o: undefined reference to symbol 'exp@@GLIBC_2.29'
/usr/bin/ld: /usr/lib/libm.so.6: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
```

--- a/raylib/cgo_linux_sdl.go
+++ b/raylib/cgo_linux_sdl.go
@@ -4,6 +4,7 @@
 package rl
 
 /*
+#cgo linux,!angle LDFLAGS: -lm
 #cgo linux CFLAGS: -DPLATFORM_DESKTOP_SDL -Wno-stringop-overflow
 #cgo linux pkg-config: sdl2
 

